parser/src/main/java/com/linkedin/avroutil1/model/SchemaOrRef.java

```diff
@@ -62,6 +68,25 @@ public String getRef() {
         return ref;
     }
 
+    // null if ref already is a FQN or if there is no parent namespace.
+    public String getInheritedName() {
```

Comment:
just like in avro-3512, ".SomeRecord" might be legal avro syntax (and is a fullname reference to SomeRecord in the null namespace).

i think your code returns "" (empty string) for both "SomeRecord" and ".SomeRecord"